### DISCUSSION TITLE
Git push

### DISCUSSION THREAD

**Discussion by ghost** [community/community#124506] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi people this is Adithya I am practicing GitHub basic but I am getting 1 error like I can't able to push to the repository please help me in this

**Comment by davevad93** [comment]
Hi @1234Adithy ,

Could you please provide more details about the error? Such as a screenshot or the output of the error.

**Comment by ghost** [comment]
Hi,

**Comment by shinybrightstar** [comment]
Hi @ghost! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
